.circleci/config.yml

```diff
@@ -37,6 +37,13 @@ commands:
                 command: bash <(curl -s https://codecov.io/bash)
 
 jobs:
+  spm-linux-job:
+    docker:
+      - image: swift:5.3
+    steps:
+      - checkout
+      - run: swift build
```

Comment:
Would be nice if we could run `swift test` on Linux but there's a few of them that require CoreLocation and other frameworks that are not available on Linux.

Reply:
I added a `swift test` command. The tests were almost working, but they exposed a memory management bug in Polyline: raphaelmor/Polyline#71.